documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -110,12 +110,30 @@ Let's see the options for the `index_schema_management_strategy` property:
 [options="header"]
 |===============
 |Value|Definition
-|`NONE`|Indexes will not be created nor deleted.
+|`NONE`|Indexes and their mappings will not be created, deleted nor altered.
+|`VALIDATE`|Existing indexes and mappings will be checked for conflicts with Hibernate Search's metamodel. Indexes and their mappings will not be created, deleted nor altered.
 |`MERGE`|Missing indexes and mappings will be created, mappings will be updated if there are no conflicts.
 |`CREATE`|**The default**: existing indexes will not be altered, missing indexes will be created.
 |`RECREATE`|Indexed will be deleted if existing and then created. This will delete all content from the index!
 |`RECREATE_DELETE`|Similarly to `RECREATE` but will also delete the index at shutdown. Commonly used for tests.
 |===============
++
+[CAUTION]
+.Strategies in production environments
+====
+It is strongly recommended to use either `NONE` or `VALIDATE` in a production environment. `CREATE` and `CREATE_DELETE` are obviously unsuitable in this context (unless you want to reindex everything upon every startup), and `MERGE` may leave your mapping half-merged in case of conflict.
+
+To be precise, if your mapping changed in an incompatible way, such as a field having its type changed, merging may be impossible. In this case, the `MERGE` strategy will prevent Hibernate Search from starting, but it may already have successfully merged another index, making a rollback difficult at best.
```

Comment:
Couldn't we validate our capability to do a merge on all indexes first, and only start the merge process if it all looks good? This option for an half-processed MERGE operation looks nasty.